### PR TITLE
Upgrade to Kaoto 2.3.0-RC1

### PR DESCRIPTION
**Pull request by apupier**
Note: command used to upgrade the dependencies: `yarn add @kaoto/kaoto@2.3.0-RC1 @patternfly/patternfly@5.4.2 @patternfly/react-code-editor@5.4.11 @patternfly/react-core@5.4.8 @
patternfly/react-icons@5.4.2 @patternfly/react-table@5.4.9 @patternfly/react-topology@5.4.1 monaco-yaml@5.2.2 react-monaco-editor@0.56.2`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@
```
Comment (djelinek) [inline README.md:45]: do we want to change also version in badge on top? https://github.com/KaotoIO/vscode-kaoto/blob/6fc01a4a7049a30072cf7d237865ee77683c8db0/README.md?plain=1#L7

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 <p align="center">
   <a href="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-kaoto?style=for-the-badge" alt="Marketplace Version"/></a>
-  <a href="https://github.com/KaotoIO/kaoto/releases"><img alt="Kaoto UI version" src="https://img.shields.io/badge/Kaoto_UI-2.2.0-orange?style=for-the-badge"></a>
+  <a href="https://github.com/KaotoIO/kaoto/releases"><img alt="Kaoto UI version" src="https://img.shields.io/badge/Kaoto_UI-2.3.0-RC1-orange?style=for-the-badge"></a>
```
Comment (djelinek) [inline README.md:7]: not sure if I remember correctly but I think double dash will be needed here `2.3.0--RC1`

**Comment by apupier** [comment]
> > Note: command used to upgrade the dependencies: `yarn add @kaoto/kaoto@2.3.0-RC1 @patternfly/patternfly@5.4.2 @patternfly/react-code-editor@5.4.11 @patternfly/react-core@5.4.8 @ patternfly/react-icons@5.4.2 @patternfly/react-table@5.4.9 @patternfly/react-topology@5.4.1 monaco-yaml@5.2.2 react-monaco-editor@0.56.2`
> 
> What about `yarn up "@patternfly/*"`?

it would require to check that Kaoto is on the latest versions of patternfly versions. the idea is to be strictly aligned to avoid errors.

File: README.md (hunk 2 above)
Comment (apupier) [reply README.md]: effectively, updated

Comment (lordrip) [reply README.md]: Yes:

Single dash image

Double dash image

**Review by lordrip — APPROVED** [review]

**Pull request merged**